Sources/NIO/EventLoopFuture.swift

```diff
@@ -815,6 +815,28 @@ extension EventLoopFuture {
     }
 }
 
+extension EventLoopFuture {
+    /// Returns a new `EventLoopFuture` that fires only when this `EventLoopFuture` and all the provided futures complete.
```

Comment:
This description doesn't really adequately capture what this method is for. It's not *wrong* at all, it just misses what this method is actually for. Mind rewording it?